plugins/signalmonitor/signalhistoryview.cpp

```diff
@@ -58,11 +58,11 @@ SignalHistoryView::SignalHistoryView(QWidget *parent)
     connect(m_eventDelegate, SIGNAL(visibleIntervalChanged(qint64)),
             this, SIGNAL(delegateVisibleIntervalChanged(qint64)));
     connect(m_eventDelegate, SIGNAL(visibleOffsetChanged(qint64)),
-            this, SLOT(eventDelegateChanged()));
+            this, SLOT(eventDelegateChanged()), Qt::QueuedConnection);
```

Comment:
Hm indeed I though identical queued event would ALL be compressed, but from reading the code at https://code.woboq.org/qt5/qtbase/src/corelib/kernel/qcoreapplication.cpp.html#1442 it seems very few are in reality.
Thanks for pointing this out!